src/components/frame/Extensions/pages/ExtensionDetailsPage/hooks.ts

```diff
@@ -0,0 +1,59 @@
+import { useCallback } from 'react';
+
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import useExtensionData, { ExtensionMethods } from '~hooks/useExtensionData.ts';
+
+import {
+  waitForColonyPermissions,
+  waitForDbAfterExtensionAction,
+} from './utils.tsx';
+
+export const useGetExtensionsViews = async () => {
```

Comment:
This will need to be refactored at some point to pull data from our DB rather than from that subgraph endpoint